ActiveRecord/Compiler/src/CodeGenerator.cpp

```diff
@@ -68,13 +68,13 @@ void CodeGenerator::writeHeaderComment(const std::string& fileName) const
 
 void CodeGenerator::writeInclude(const std::string& nameSpace, const std::string& name) const
 {
-	_stream << "#include \"";
+	_stream << "#include <";
```

Comment:
@rastocny why this?

Reply:
Optimization of compilation process and cleanness in dependencies. `<>` are used for includes from external libraries and compiler shall look for included files in external includes (in MSVC Additional includes directory). `""` are used for internal includes, that shall have relative path from current file. When "internally" included file is not found, a fallback to external includes is applied.
- https://learn.microsoft.com/en-us/cpp/preprocessor/hash-include-directive-c-cpp?view=msvc-170
- https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html

The generator generates source code files that are externally depended to `PoCo` libraries.

Reply:
I'll leave this to @obiltschnig to decide